### ISSUE TITLE
Make a Mad Dash

### ISSUE THREAD

**Issue by Smolations** [scotthmurray/strata-d3-tutorial#1]
Your d3 cheat sheet is fantastic! This issue is not a bug, but rather a feature request. Almost all of the devs at my company use [Dash](http://kapeli.com/dash) to reference documentation during development. It would be awesome if you provided your cheat sheet as a user-contributed docset for that piece of software. I know it would help me out quite a bit!

:smile:

**Comment by scotthmurray** [comment]
Thanks for that, @Smolations. I haven't used Dash before, but it looks like there already exists a d3.js doc set.  Have you tried that out?  If so, can you help me understand the benefit to making an alternate D3 package for Dash?  Thanks!

**Comment by Smolations** [comment]
Sure!  Dash has two main types of documentation: docsets and cheatsheets. Docsets are the FULL documentation for all methods, etc. that is usually found on the software website. Cheat sheets are, well, I think you know.  =]

I was requesting specifically to add the cheat sheet to the list of user-contributed cheat sheets. It's way less work than adding a full docset, I assure you. :laughing:

**Comment by scotthmurray** [comment]
Ah, thanks. I will look into this. I'm in the process of revising my book for a 2nd edition, and it could make sense to release a Dash-ified version of the cheat sheet as part of that code package.

**Comment by Smolations** [comment]
Awesome, thanks for your consideration! :+1:

**Comment by scotthmurray** [comment]
@Smolations Have you seen this?

http://devdocs.io/

**Comment by Smolations** [comment]
@alignedleft I have not, but it doesn't quite solve my problem either. While it may have docsets Dash is missing, it shares much of the documentation Dash provides. For me, the two main reasons to purchase Dash were:
1. **All documentation in one place, and no browser required.** One app, one set of keyboard shortcuts, one source of truth. I was simply tired of having multiple browser tabs open for documentation.
2. **Offline browsing**. Unless I do some strategic configuration, that website is only available while connected to the internet. Since Dash uses documentation bundles (_docsets_), they can be viewed offline, and only need an internet connection for updates.

I do appreciate the suggestion, though!  :smile:

**Comment by scotthmurray** [comment]
@Smolations Yes, of course! I was just thinking it could be a nice alternative (for different contexts, perhaps).
